etl_pipeline/outages/load_outages.py

```diff
@@ -0,0 +1,216 @@
+"""
+Load outage data into RDS
+
+- Reads DB creds from .env (DB_HOST, DB_PORT, DB_DB, DB_USER, DB_PASSWORD)
+- Upserts `outage`
+- Inserts `postcode` (ignore duplicates)
+- Inserts `outage_postcode_link` by staging (outage_id, postcode) into a TEMP table
+  and resolving postcode_id
+"""
+
+from __future__ import annotations
+
+import json
+import os
```

Comment:
Could be more specific with what we are using from imports. For instance,

```suggestion
from os import environ as ENV 
```

same could apply for psycopg2, and json.

Reply:
Made the changes left json as is, as its all I need